verilog/tools/ls/verilog-language-server_test.cc

```diff
@@ -580,6 +580,19 @@ std::string DefinitionRequest(absl::string_view file, int id, int line,
   return formattingrequest.dump();
 }
 
+// Creates a textDocument/definition request
+std::string ReferencesRequest(absl::string_view file, int id, int line,
```

Comment:
Can you assemble these from the verible::lsp:: structs ? The json is quickly getting pretty hard to read.
There is verible::lsp::Reference which we can use.